wagon.py

```diff
@@ -259,8 +266,8 @@ def install_package(package,
 
 
 def _get_downloaded_wheels(path):
-    return \
-        [filename for filename in os.listdir(path) if filename.endswith('whl')]
+    return [filename for filename in os.listdir(path)
+            if os.path.splitext(filename)[1]]
```

Comment:
adding to Dan's note, i'm guessing this should have been `if os.path.splitext(filename)[1].lower() == '.whl'`